lib/use-dom-columns.js

```diff
@@ -116,6 +117,8 @@ const columnSymbol = Symbol('column'),
 			// treenode columns
 			ownerTree: column.ownerTree,
 			keyProperty: column.keyProperty,
+			hideFromRoot: column.hideFromRoot,
+			showMaxNodes: column.showMaxNodes,
 
```

Comment:
To avoid listing all the possible properties of all columns in one place we should refactor this a bit to call a optional function implementation `getConfig` on each column. 

```suggestion
...column.getConfig?.(column),
```

In that case in treenode columns we can implement this: 
```
 getConfig(column) {
 return {
limit: column.limit,
hideFromRoot: column.hideFromRoot
....etc
}
}

```

Reply:
ok, pushed change.